adkintunmobile-server/populate/Dockerfile

```diff
@@ -0,0 +1,51 @@
+## Header
+FROM python:3.5.1-slim
+MAINTAINER Boris Romero <boris@niclabs.cl>
+
+
+## Dependencies
+# install apt dependencies
+RUN apt-get update && \
+	apt-get install -y \
+		gcc \
+		git \
+		libpq-dev \
+		make \
+		wget
+
+# clone project
+RUN mkdir /adk && \
+	cd /adk && \
+	git clone https://github.com/niclabs/AdkintunMobile-Server.git && \
+	cd AdkintunMobile-Server && \
+	git checkout 451e2637dd7bf15d0574b49d2397e90f548ef6ab 
```

Comment:
Yes! I remember now we spoke this when we create the dockerfiles. Thanks @franchoco